setup-scripts/skynet-nginx.conf

```diff
@@ -1,3 +1,7 @@
+limit_req_zone $binary_remote_addr zone=stats:10m rate=10r/m;
```

Comment:
10r/m  feels like a very constricting rate, is there any reason we're being so conservative? Could we bump this up to 1r/s?

Reply:
oh I'm sorry I missed the zone, I think I would drop this down to 1r/m in that case :D